packages/swingset-runner/demo/vaultPerfTest/vaultPerfTest-config.json

```diff
@@ -0,0 +1,211 @@
+{
+  "$comment": "This SwingSet config file (see loadSwingsetConfigFile) is designed to bring up vaults test networks in an automated fashion. It includes coreProposals to start vaults. Testing facilities are limited to an initialPrice for ATOM and addresses with known keys.",
```

Comment:
This looks like it was copied from an existing config file. Is it intended to be maintained independently? Or when the original changes, is this supposed to stay in sync? If they're supposed to stay in sync, please derive this from the other at runtime or add documentation pointers in both directions or something.

Reply:
Eventually (soon) this file will go away entirely, as it is basically a copy of the config file from `vm-config`.  It's only here because swingset-runner needs it to be where it is, which is a limitation that I intend to correct.

Reply:
I've pushed an additional commit that fixes the wart that made this file necessary (and removed the file itself as part of that).